[2025-07-30T11:11:51.204+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-30T11:11:51.309+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.transform_data manual__2025-07-30T11:11:47.518903+00:00 [queued]>
[2025-07-30T11:11:51.344+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.transform_data manual__2025-07-30T11:11:47.518903+00:00 [queued]>
[2025-07-30T11:11:51.346+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-30T11:11:51.658+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-30 11:11:47.518903+00:00
[2025-07-30T11:11:51.698+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'transform_data', 'manual__2025-07-30T11:11:47.518903+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmp15irelps']
[2025-07-30T11:11:51.722+0000] {standard_task_runner.py:91} INFO - Job 238: Subtask transform_data
[2025-07-30T11:11:51.734+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=562) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-30T11:11:51.741+0000] {standard_task_runner.py:63} INFO - Started process 564 to run task
[2025-07-30T11:11:51.842+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.transform_data manual__2025-07-30T11:11:47.518903+00:00 [running]> on host 9c353dd7cf4b
[2025-07-30T11:11:51.940+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T11:11:47.518903+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T11:11:47.518903+00:00'
[2025-07-30T11:11:51.941+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-30T11:11:52.003+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-30T11:11:52.004+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-30T11:11:52.024+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=transform_data, execution_date=20250730T111147, start_date=20250730T111151, end_date=20250730T111152
[2025-07-30T11:11:52.112+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-30T11:11:52.200+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-30T11:11:52.213+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
